.gitignore

```diff
@@ -5,6 +5,10 @@
 
 # vcstool packages
 /src/*
+!/src/integration_tests
+/src/integration_tests/integration_tests/ros_dtypes.py
+/src/integration_tests/config
+!/src/integration_tests/generate_configs.sh
```

Comment:
I don't see this file. Is it because it is the wrong filepath, it's in the config directory right

Reply:
Fixed.